### PR TITLE
Update trigger list in project notifications

### PR DESCRIPTION
**Pull request by dhlevi**
Adding 60 new trigger values for project notifications. Changing select box to multiselect with autofill.

Project Notification datatype is a comma separated string, but we can update the data source to use an array if needed (will require a migration).

See ticket [EE-977](https://bcmines.atlassian.net/browse/EE-977)

Note that this also covers changes required to Nature in [EE-991](https://bcmines.atlassian.net/browse/EE-991)

### CODE REVIEW

**Comment by dhlevi** [comment]
Yeah yeah, cleaned up the extra space! Should be building now

File: src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.ts [hunk 1]
```diff
@@ -170,7 +178,20 @@ export class AddEditProjectNotificationComponent implements OnInit, OnDestroy {
...
+    this.triggers = [];
+    let dataTriggers =  data.trigger.split(',');
+
+    Constants.NOTIFICATION_TRIGGERS.forEach(trigger => {
+      if (dataTriggers.includes(trigger.name)) {
+        this.triggers.push(trigger);
+      }
+    });
+
+    const natureDisabled = !data.trigger.split(',').includes('Greenhouse Gases (modification)') && !data.trigger.split(',').includes('Greenhouse Gases (new project)');
```
Comment (a1astair) [inline src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.ts:194]: Could we put these strings in the config file with all the other strings for the nature field?

Comment (dhlevi) [reply src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.ts:194]: These aren't nature values, they're trigger values. I can pull them from the constants as well, but I'd still need to search for the specific name at some point so it'll be hard coded here, or hard-coded elsewhere

Comment (a1astair) [reply src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.ts:194]: yeah fair enough!

**Pull request merged**